core/src/main/java/com/airbnb/aerosolve/core/models/NDTreeModel.java

```diff
@@ -42,6 +43,32 @@ public NDTreeModel(List<NDTreeNode> nodeList) {
     this(nodeList.toArray(new NDTreeNode[nodeList.size()]));
   }
 
+  public double stdDivByMinLeafNodesCount() {
+    // Get a DescriptiveStatistics instance
+    DescriptiveStatistics stats = new DescriptiveStatistics();
+
+    for (NDTreeNode n : nodes) {
+      if (isLeaf(n)) {
+        stats.addValue(n.count);
+      }
+    }
+    double std = stats.getStandardDeviation();
```

Comment:
standard deviation is easy to implement. If std is the only thing we need from the math3 library, perhaps we can just implement ourselves without adding the dependency on the external package.